doc/classes/EditorSettings.xml

```diff
@@ -1044,6 +1044,9 @@
 			The indentation style to use (tabs or spaces).
 			[b]Note:[/b] The [url=$DOCS_URL/tutorials/scripting/gdscript/gdscript_styleguide.html]GDScript style guide[/url] recommends using tabs for indentation. It is advised to change this setting only if you need to work on a project that currently uses spaces for indentation.
 		</member>
+		<member name="text_editor/behavior/navigation/custom_word_separators" type="String" setter="" getter="">
+			If [member text_editor/behavior/navigation/use_custom_word_separators] is enabled, use these characters as word delimiters. Enter characters without spaces.
```

Comment:
`true` for consistency. Also elaborating on this setting a bit, as "without spaces" is a bit vague (_what if I want the space to be a delimiter, too?_).
```suggestion
			The characters to consider as word delimiters, if [member text_editor/behavior/navigation/use_custom_word_separators] is [code]true[/code]. The characters should be defined without separation, for example [code]`~!@#$%^&*()-=+[{]}|;:'\",.<>/?[/code].
```
This makes me wonder, is the distinction between "built-in" and "custom" separators even necessary?
If the editor setting was just called `text_editor/behavior/navigation/word_separators` and it contained the defaults... by default, I feel like it would be more natural.

Reply:
The default include a lot of non-visual character (various spacial spaces), setting and editing it in the editor settings value to it would be a mess.

Reply:
Yes, that would be a lot. I also reduced the examples to three characters in the documentation: [code]`~![/code]

The characters to consider as word delimiters if [member text_editor/behavior/navigation/use_custom_word_separators] is [code]true[/code]. The characters should be defined without separation, for example [code]`~!@#$[/code].